_examples/apache-zeppelin-example.adoc

```diff
@@ -0,0 +1,81 @@
+= Apache Zeppelin  Example
+:page-link: apache-zeppelin-example
+:page-weight: 100
+:page-labels: [Notebook, Zeppelin, Spark, R, Pyspark, Scala, Java]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This is an example of how to use Apache Zeppelin
+:page-project_links: ["https://github.com/rimolive/zeppelin-openshift", "https://github.com/rimolive/zeppelin-notebooks"]
+
+[[introduction]]
+== Introduction
+
+Apache Zeppelin is an alternative to project http://jupyter.org/[Jupyter] to provide 
+a notebook to run analysis over a data and plot graphs in order to show to end users
+an orverview about the data.
```

Comment:
s/orverview/overview/

also, i think this sentence should mention more than just an overview of the data. maybe say something about how the notebook allows users to explore and interact with data?